src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -19,6 +19,17 @@ abstract class PipelineTargets(config: Config) {
       "writeOpsMetrics, lastOptimizedTS, Pipeline_SnapTS, primordialDateString").split(", ")
   )
 
+  val apiEventsTarget: PipelineTable = PipelineTable(
+    name = "api_events_meta",
+    _keys = Array("organization_id", "moduleId", "Overwatch_RunID"),
+    config = config,
+    partitionBy = Array("organization_id"),
+    incrementalColumns = Array("Pipeline_SnapTS"),
+    statsColumns = ("organization_id, workspace_name, moduleID, moduleName, runStartTS, runEndTS, " +
+      "fromTS, untilTS, status, " +
+      "writeOpsMetrics, lastOptimizedTS, Pipeline_SnapTS, primordialDateString").split(", ")
+  )
```

Comment:
these aren't right

Reply:
need to understand more about statsColumn.